task/sql-tasks.js

```diff
@@ -281,7 +461,22 @@ async function task_1_21(db) {
  * @return {array}
  */
 async function task_1_22(db) {

```

Comment:
the query is too slow, not sure why. Might be because of selecting from multiple tables (this is an old approach, dont use it)

Reply:
@AlexMokin Hi! Thank you for your comments! I have resolved all the conversations by new commits. Could you please check the new version? (travis-ci build https://travis-ci.org/github/AlinaNH/db-assignments/builds/722776674)

Reply:
You need to click "ready for review". It'll be added to the list "for review". Right now PR marked as draft